CI/scripts/algorithm_names_map.dat

```diff
@@ -0,0 +1,52 @@
+# #############################################################################
+# Certifier Framework Encryption algorigthm names mapping
+# Specify the way hard-coded strings will be remapped based on some
+# organization.
+# #############################################################################
+#
+# enc_method:Encryption algorithms supported.
+#
+enc_method:"aes-128"
+enc_method:"aes-256"
```

Comment:
I prefer to use `Enc_method` as prefix, so that the global variable appears with 1st char `U`pper-case.

It's just a small convention to let the user 'know' quickly that this is a global variable / string-constant.

Reply:
Also, another thought I had was to "name space" these constants into Certifier Frameworks' universe.

What about using consistently "Cf_" prefix; so it will be stuff like `Cf_enc_method_aes_128` or `CF_enc_method_aes_256` ... and so on.

A bit long, but may be in our protection to avoid name space clashes in future. 
no?